Scripts/developer_scripts/add_license_in_pkg_header.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Insert in all the header file of a package the include directive
+# "#include <CGAL/license/${package_name}.h" if it is not already there
+# There are two arguments to the script as some packages are split into
+# subdirectories (Polyhedron_IO, Algebraic_kernel_for_circles, ...)
+
+from sys import argv
+import os
+from os.path import join
+import codecs
+import re
+
+def add_license_include_in_file(package_name, fname):
+  # first, see if the include directive is already there
+  with codecs.open(fname, encoding='utf-8') as f:
+    if any(re.search("CGAL/license/"+package_name+".h", line) for line in f):
+      return # instruction already there
+
+  # pattern is in the file, so perform replace operation.
+  inserted = False
+  with codecs.open(fname, encoding='utf-8') as f:
+    out_fname = fname + ".tmp"
+    out = codecs.open(out_fname, "w", encoding='utf-8')
+    for line in f:
+      if not inserted and re.search("#\s*define", line):
```

Comment:
Here I would search for a more explicit `#\s*define\s*CGAL.*_H`. Otherwise special files like `Kernel_23/include/CGAL/Kernel/interface_macros.h` will get corrupted.